codegen/go-composite-schema/main.go

```diff
@@ -50,11 +50,13 @@ go-composite-schema to correctly determine the correct package name.
 
 
   Usage:
-    go-composite-schema [--required] PROPERTY INPUT-FILE OUTPUT-FILE
+    go-composite-schema [--required] [--unexported] PROPERTY INPUT-FILE OUTPUT-FILE
     go-composite-schema -h|--help
     go-composite-schema --version
 
   Options:
+    --required             Implies the given composite schema defines a required property.
+    --unexported           Generate an unexported type for the given composite schema.
```

Comment:
We could just make this implied from the `title` property... Note: I still propose introducing a `TypeName` property in the JSON schema, so `title` only is used as a fallback.